loops-and-arrays.js

```diff
@@ -23,6 +28,19 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // ==========================================
 
 const numbers = [2, 4, 5, 29, 38];
+const modifiedNumbers = [ ];
+
+for (let i = 0; i <numbers.length; i++) {
+    if (numbers[i] % 2 === 0) {
+        modifiedNumbers.push(numbers[i] * 2);
+    } else {
+        modifiedNumbers.push(numbers[i] * 3);
+    }
+}
+
+console.log(modifiedNumbers);
```

Comment:
Keurig! Het was niet persé nodig om een extra variabele aan te maken. 

for (let i = 0; i < numbers.length; i++) {
  if (numbers[i] % 2 === 0) {
    numbers[i] = numbers[i] * 2;
  }
  if (numbers[i] % 2 !== 0) {
    numbers[i] = numbers[i] * 3;
  }

HIermee overschrijf je de huidige waardes in de array.